### ISSUE TITLE
Composed UVL feature model does not update when a referenced submodel is changed

### ISSUE THREAD

**Issue by SundermannC** [FeatureIDE/FeatureIDE#1134]
When using the composition mechanism of UVL, the composed feature model does not update after a referenced feature model is changed. After updating a submodel, the composed feature model should be notified and adjusted according to the changes.

**Comment by jellehierck** [comment]
I agree that feature models should be updated automatically when referenced submodels change.

In the mean time, is there currently any manual way to trigger a parent model to update if a referenced submodel has changed, without deleting and re-importing the submodel?

**Comment by SundermannC** [comment]
Thanks a lot for your comment and providing an answer. I agree that the most convenient solution currently is re-opening the feature model editor of the composed feature model.
